src/Illuminate/Database/Schema/Builder.php

```diff
@@ -378,9 +387,31 @@ public function enableForeignKeyConstraints()
      */
     public function disableForeignKeyConstraints()
     {
-        return $this->connection->statement(
+        $this->hasForeignKeyConstraints = $this->connection->statement(
```

Comment:
Since you're assigning the value I feel like you'd might as well return in one line?

```php
return $this->hasForeignKeyConstraints = $this->connection->statement('...');
```

Reply:
Fair enough, that makes sense, however taylor stated it may be returning true all the time, so I need to investigate if this logic is correct because I know that `connection->statement` returns a boolean, however I'm not sure what that boolean means.